superset-frontend/src/views/CRUD/data/database/DatabaseList.tsx

```diff
@@ -437,10 +439,11 @@ function DatabaseList({ addDangerToast, addSuccessToast }: DatabaseListProps) {
       {databaseCurrentlyDeleting && (
         <DeleteModal
           description={t(
-            'The database %s is linked to %s charts that appear on %s dashboards. Are you sure you want to continue? Deleting the database will break those objects.',
+            'The database %s is linked to %s charts that appear on %s dashboards and users have %s SQL Lab tabs using this database open. Are you sure you want to continue? Deleting the database will break those objects.',
```

Comment:
It would be nice to skip this message if all the counts are 0, something like:

```js
const message = (
  databaseCurrentlyDeleting.chart_count +
  databaseCurrentlyDeleting.dashboard_count +
  databaseCurrentlyDeleting.sqllab_tab_count) === 0
  ? 'There are no charts, dashboards or queries associated with this database, it should be safe to delete.'
  : 'The database %s is linked to ...';
```